<launch>
	<arg name="imagestream" value="/cam0/image_rect"  />
	<!-- Available methods: "VOCUS2", "Itti", "Hou"-->
	<arg name="saliency_algorithm" value="VOCUS2" />

	<node name="saliencymap$(arg saliency_algorithm)" pkg="visual_saliency" type="SaliencyDetection" output="screen">
		<remap from="/saliency_detection/rgbimage_in" to="$(arg imagestream)" />
		<!-- <remap from="/saliency/image" to="/saliency/image" /> -->
		<param name="saliency_algorithm" type="string" value="$(arg saliency_algorithm)" />
		<param name="image_resize_percent" type="double" value="0.25"/>
		<param name="threshold_percent" type="double" value="0.05" />
		<param name="topcut_enable" type="bool" value="false" />
		<param name="topcut_pixels" type="int" value="0" />
		<param name="time_show" type="bool" value="true" />
	</node>

	<arg name="rviz" default="true" />
	<group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find visual_saliency)/rviz_cfg/config.rviz" />
  </group>

</launch>
